Polyhedron/demo/Polyhedron/Scene_item.h

```diff
@@ -138,17 +138,18 @@ class SCENE_ITEM_EXPORT Scene_item : public QObject {
 public Q_SLOTS:
   // Call that once you have finished changing something in the item
   // (either the properties or internal data)
-  virtual void changed();
+  virtual void invalidate_buffers();
   virtual void contextual_changed(){}
 
   // Setters for the four basic properties
-  virtual void setColor(QColor c) { color_ = c; changed(); }
+  virtual void setColor(QColor c) { color_ = c; invalidate_buffers(); }
   void setRbgColor(int r, int g, int b) { setColor(QColor(r, g, b)); }
```

Comment:
if the buffer are invalidated here, there is no need to do it in `Scene::setData`